### DISCUSSION TITLE
Change animation for navigation between pages

### DISCUSSION THREAD

**Discussion by Skvorecz** [MvvmCross/MvvmCross#4099] [Q&A]
Hello.
My application has several pages. To navigate between pages i'm using MvxNavigationService.Navigate method.
When i'm navigating to a new page it pushes from the bottom of a screen.
So my issue is i want to change that animation, for example make new page move from left border.
Is there any suggestions?

**Comment by Cheesebaron** [comment]
Which platform? Have you looked into the PresentationAttribute properties? On Android and iOS you can set animations there.
